audify/readers/ebook.py

```
@@ -99,25 +104,221 @@ def __init__(
+        cannot be used (empty, unparseable, or mismatched with the spine).
         """
-        chapters = []
+        toc_item_names = self._build_toc_item_name_set()
+        if not toc_item_names:
+            return []
+
+        chapters: list[str] = []
+        current_group: list = []
+        matches_found = 0
+
+        for spine_id, linear in self.book.spine:
```

Ruff (0.15.11)

[warning] Loop control variable `linear` not used within loop body

Rename unused `linear` to `_linear`

(B007)